### PR TITLE
feat: 🎸 Expose SS58 format

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
#### JIRA Link

https://polymath.atlassian.net/browse/DA-247

### CODE REVIEW

**Review by polymath-eric — COMMENTED** [review]
I am surprised sonar cloud said this was OK with 0% test coverage. I wonder if its a bug.

I know its one line with no logic, but can you add a test case for this.

File: src/__tests__/Network.ts [hunk 1]
```diff
@@ -77,6 +77,18 @@ describe('Network Class', () => {
     });
   });
 
+  describe('method: getSs58Format', () => {
+    it("should return the chain's SS58 format", async () => {
+      const ss58Format = new BigNumber(42);
+
+      dsMockUtils.configureMocks({ contextOptions: { ss58Format } });
+
+      const result = await network.getSs58Format();
```
Comment (polymath-eric) [inline src/__tests__/Network.ts:86]:
```suggestion
      const result = network.getSs58Format();
```

Not async (sonar says its a smell)

**Review by polymath-eric — APPROVED** [review]
LGTM, just sonar smell

File: src/__tests__/Network.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/__tests__/Network.ts]: Oops. My bad. File remained unsaved. Pushed up a patch for it

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**